src/main/java/appeng/items/parts/ItemPart.java

```diff
@@ -297,22 +298,40 @@ public String getUnlocalizedGroupName( final Set<ItemStack> others, final ItemSt
 							group = true;
 						}
 						break;
+					case FLUID_IMPORT_BUS:
+						importBusFluids = true;
+						if( u == pt )
+						{
+							group = true;
+						}
+						break;
 					case EXPORT_BUS:
 						exportBus = true;
 						if( u == pt )
 						{
 							group = true;
 						}
 						break;
+					case FLUID_EXPORT_BUS:
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce this switch case number of lines from 6 to at most 5, for example by extracting code into methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1151)